### ISSUE TITLE
Bounding boxes are not correct after exporting to coco

### ISSUE THREAD

**Issue by jokober** [scalabel/scalabel#527]
**Describe the bug**
I just realized that the bounding boxes created with scalabel and then exported to coco are not correct anymore. They are not shown at the correct spot anymore.

**To Reproduce**
1. Download data
`wget  -O scalabel_formatted.json http://domain:81/getExport?project_name=fancy_project`
3. Convert scalabel format to coco
`python3 -m scalabel.label.to_coco --input scalabel_formatted.json -o coco.json -m box_track --config /local_dir/scalabel/config.json`
4. Check resulting coco file using tools like fiftyone

**Expected behavior**
Export the bounding boxes correctly

**Screenshots**
Sceenshot of scalabel:
![image](https://github.com/scalabel/scalabel/assets/4704377/6f50ded6-4de9-4cba-bdcc-c9380ed3deb4)

Screenshot of fiftyone:
![Screenshot from 2023-08-28 18-14-34](https://github.com/scalabel/scalabel/assets/4704377/0fe9b86c-b02e-4abb-961d-fba36c9eee5a)

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser Firefox

**Comment by jokober** [comment]
I just realized that the y coordinate seems to be correct always. Its only the x coordinates which are wrong

**Comment by jokober** [comment]
I also found out the vertical displacement becomes higher the further on the right the bounding box is. On the left side the vertical displacement is minimal...

**Comment by jokober** [comment]
I tested two more things in order to narrow down the problem::
1. First I reimported the downloaded scalabel formatted project back into scalabel. The bboxes are shown correctly in scalabel.
2. I converted the exported scalabel formatted project to coco and then from coco back to scalabel formatted project. Then I reimported the project back to scalabel. The bounding boxes are shown correctly in scalabel.
